core/state_transition.go

```diff
@@ -436,20 +450,80 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		return nil, fmt.Errorf("%w: code size %v limit %v", ErrMaxInitCodeSizeExceeded, len(msg.Data), params.MaxInitCodeSize)
 	}
 
+	// Verify authorization list is not empty.
```

Comment:
```suggestion
	// Verify a non-nil authorization list is not empty.
```

Reply:
Okay maybe a bad wording, but I think you get what I mean. This message should convey that the authorization list could either be non-existent or have at least one entry

Reply:
updated